acquia_lift.admin.inc

```diff
@@ -448,7 +493,7 @@ function acquia_lift_report_custom($form, &$form_state, $agent, $agent_data, $op
   $reports = $agent->buildCampaignReports($options);
   if (!($reports['#has_data'])) {
     // This campaign hasn't been shown yet, so there is no data for reporting.
-    return array(
+    $form = array(
       'no_report' => array(
         '#markup' => t('This personalization does not yet contain information about your visitors\' website interactions. This situation generally occurs for personalizations that have either just been started or do not generate much website traffic.'),
       ),
```

Comment:
Should return here?  Looks confusing to see this message and then see the big empty report.